test/L2/L2Reward.t.sol

```diff
@@ -113,7 +113,7 @@ contract L2RewardTest is Test {
         l2Staking.addCreator(address(l2Reward));
     }
 
-    function test_initialize() public {
+    function test_initialize() public view {
```

Comment:
Just to be sure, why do we need to make those functions of type `view` in all unit test files?

Reply:
After putting Foundry version back to `Nightly` inside `pr.yaml` file, compiler thrown warnings about adding `view` to some of test functions because they are not changing test contract state.